conformance/fhir-ig-davinci-hrex/src/main/resources/hl7/fhir/us/davinci-hrex/package/xml/StructureDefinition-hrex-coverage.sch

```diff
@@ -0,0 +1 @@
+C:\GitHub\hl7.fhir.us.davinci-hrex#0.2.0\temp\pages\StructureDefinition-hrex-coverage.sch
```

Comment:
I don't think we should include the schematron files

Reply:
removed

Reply:
hmm, I'm still seeing these in the latest.  maybe you forgot to push it?

Reply:
I actually removed from PDEX not HREX

Reply:
Fixed